src/assets/translations/messages-en.json

```
@@ -418,7 +439,7 @@
       },
       "accounts": "Accounts",
       "stakingVaults": "Staking & Vaults",
-      "stakingBody": "Earn passive income by staking or deposting your assets into a vault.",
+      "stakingBody": "Earn passive income by staking or deporting your assets into a vault.",
```

Should it not be depositing?
```suggestion
      "stakingBody": "Earn passive income by staking or depositing your assets into a vault.",
```

---

Yup. I didn't verify the spellcheck change.